### ISSUE TITLE
add some kind of in-progress tag

### ISSUE THREAD

**Issue by sjz** [tanepiper/sublime-todomanager#4]
Ala [Kanban](http://www.kanban101.com)

Could be as simple as adding the text `doing:` - this would be in keeping with the todo.txt format of addonkey:value but as it's a boolean there's really no need for a value bit, but I guess it could also be `doing:true/false`

Hope that makes sense.

I still plan to help out btw..eventually..

**Comment by tanepiper** [comment]
Yes, I'm going to change the way the additional features like line numbers work to support this format
